README.md

```diff
@@ -219,6 +219,59 @@ GET /whiteboard_tags/
 }
 ```
 
+## Troubleshooting
+
+### Bugzilla tickets are not showing up as issues on Jira
+
+Checklist:
+
+- The Bugzilla Webhook is setup to be executed for your project
+- The Bugzilla Webhook is still **enabled**
+- The whiteboard tag is listed in `GET /whiteboard_tags/`
+- The Jira project is listed in `GET /jira_projects/`
```

Comment:
Would an explicit action be easier: `open https://jbi.services.mozilla.com/powered_by_jbi/` and check if your project is listed there.